### ISSUE TITLE
A few more touchups...

### ISSUE THREAD

**Issue by bryanwillis** [ThematoSoup/THA-Hooks-Interface#1]
Hey I posted on the wordpress support page just noticed you were on github so thought you'd rather have it here since you mentioned you hadn't subscribed to the WP plugin.

Anyway, here's two more fixes for the admin screen. This is for when the checkboxes aren't checked:

```
public function field_cb( $args ) {
        $hooks_group = $args[0];
        $hook_name = $args[1];
        $hook_description = $args[2];
        $output_field_name = 'tha_hooks_interface_' . $hooks_group . '[' . $hook_name . '][output]';
        $php_field_name = 'tha_hooks_interface_' . $hooks_group . '[' . $hook_name . '][php]';
        $shortcode_field_name = 'tha_hooks_interface_' . $hooks_group . '[' . $hook_name . '][shortcode]';
        $tha_interface_settings = get_option( 'tha_hooks_interface_' . $hooks_group );
        $shortcode_checked = isset( $tha_interface_settings[ $hook_name ]['shortcode'] ) ? $tha_interface_settings[ $hook_name ]['shortcode'] : '';
        $php_checked = isset( $tha_interface_settings[ $hook_name ]['php'] ) ? $tha_interface_settings[ $hook_name ]['php'] : '';
        ?>
        <p><?php echo $hook_description; ?></p>
        <p>
        <textarea style="font-family:monospace" rows="10" class="widefat" name="<?php echo $output_field_name; ?>" id="<?php echo $output_field_name; ?>"><?php echo htmlentities( $tha_interface_settings[ $hook_name ]['output'], ENT_QUOTES, 'UTF-8' ); ?></textarea>
        </p>
        <?php if ( current_user_can( 'unfiltered_html' ) ) : ?>
        <p>
        <label for="<?php echo $php_field_name; ?>">
            <input type="checkbox" name="<?php echo $php_field_name; ?>" id="<?php echo $php_field_name; ?>" value="1" <?php checked( $php_checked, 1 ); ?>/>
            <?php _e( 'Execute PHP in this hook (must be enclodes in opening and closing PHP tags)', $this->plugin_slug ); ?>
        </label>
        </p>
        <?php endif; ?>
        <p>
        <label for="<?php echo $shortcode_field_name; ?>">
            <input type="checkbox" name="<?php echo $shortcode_field_name; ?>" id="<?php echo $shortcode_field_name; ?>" value="1" <?php checked( $shortcode_checked, 1 ); ?> />
            <?php _e( 'Run shortcodes in this hook', $this->plugin_slug ); ?>
        </label>
        </p>
    <?php }
```

**Comment by slobodan** [comment]
Yeah, I missed it there, sorry. Will look into this as soon as I can, but since I'll be moving this weekend it might have to wait until next week.

Really appreciate it though.

**Comment by slobodan** [comment]
Fixed in 1.1, thanks again!
